src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -160,7 +162,6 @@ public void registerVillageCreationHandler(IVillageCreationHandler handler)
      * @param defaultSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
```

Comment:
now just the one method that takes an int and a default texture is left, that one is currently used on line 51 of the RenderVillager class, but shouldn't be needed by any mods.